### ISSUE TITLE
my training loss didn't changed, why?

### ISSUE THREAD

**Issue by suyanzhou626** [meetshah1995/pytorch-semseg#134]
![image](https://user-images.githubusercontent.com/37359271/46134046-8d743c00-c274-11e8-8014-b375d59807f7.png)
like this

**Comment by nileshkumar0726** [comment]
I am also having a similar issue, did you find the reason or solution?

**Comment by pvbhanuteja** [comment]
set less learning rate
